admin_manual/configuration/server/occ_command.rst

```diff
@@ -1249,10 +1249,23 @@ Use the ``--enable`` option to turn on logging. Use ``--file`` to set a differen
 Set your rotation by log file size in bytes with ``--rotate-size``; 0 disables rotation. 
 ``log:manage`` sets your logging backend, log level, and timezone. 
 The defaults are ``owncloud``, ``Warning``, and ``UTC``. 
+
+Log level can be adjusted by entering the number or the name:
+
+::
+
+   sudo -u www-data php occ log:manage --level 4
+   sudo -u www-data php occ log:manage --level error
+
+You can also choose debug, info, warning, error or fatal.
+
+.. note::
+   Setting the log level to debug ( 0 ) can be used for finding the cause for an error, but should not be the standard
```

Comment:
One minor change: "cause for an error" should be "cause of an error". Otherwise, lgtm.

Reply:
you are fast!

Reply:
oh, you already changed it, I did not knew reviewers can change the pr

Reply:
I've not changed it mate. Just made a comment.